src/v2/cookbook/packaging-sfc-for-npm.md

```diff
@@ -124,96 +124,103 @@ There is no need to write your module multiple times. It is possible to prepare
 
-### How do I configure the Rollup build?
+### Comment configurer un build avec Rollup ?
 
-With the package.json `scripts` section ready and the SFC wrapper in place, all that is left is to ensure Rollup is properly configured. Fortunately, this can be done with a small 16 line rollup.config.js file:
+Avec notre package.json contenant la section `script` prête à démarrer et la configuration de notre [Composant](https://fr.vuejs.org/v2/guide/single-file-components.html) mise en place via notre wrapper, il ne reste plus qu'à s'assurer que Rollup est correctement configuré. Heureusement, cela peut être fait avec un petit fichier rollup.config.js de 19 lignes déjà prêtes:
 
 ```js
-import commonjs from '@rollup/plugin-commonjs'; // Convert CommonJS modules to ES6
-import vue from 'rollup-plugin-vue'; // Handle .vue SFC files
-import buble from '@rollup/plugin-buble'; // Transpile/polyfill with reasonable browser support
+import commonjs from '@rollup/plugin-commonjs'; // Converti les modules CommonJS en ES6
+import vue from 'rollup-plugin-vue'; // Gère les composants .vue
+import bubble from '@rollup/plugin-buble'; // Permet un polyfill de notre code pour un meilleur support sur les navigateurs
+
 export default {
-    input: 'src/wrapper.js', // Path relative to package.json
-    output: {
-        name: 'MyComponent',
-        exports: 'named',
-    },
-    plugins: [
-        commonjs(),
-        vue({
-            css: true, // Dynamically inject css as a <style> tag
-            compileTemplate: true, // Explicitly convert template to render function
-        }),
-        buble(), // Transpile to ES5
-    ],
+  input: 'src/wrapper.js', // Fichier relatif en partant de notre fichier package.json
+  output: {
+    name: 'MyComponent',
+    exports: 'named',
+  },
+  plugins: [
+    commonjs(),
+    vue({
+      css: true, // Injecte dynamiquement notre CSS dans une balise <style>
+      compileTemplate: true, // Converti notre template en fonction de rendu Vue
+    }),
+    bubble(), // Traduit en ES5
+  ],
 };
 ```
 
-This sample config file contains the minimum settings to package your SFC for npm. There is room for customization, such as extracting CSS to a separate file, using a CSS preprocessor, uglifying the JS output, etc.
+Cet exemple de configuration contient les paramètres minimums pour empaqueter votre [Composant](https://fr.vuejs.org/v2/guide/single-file-components.html) à destination de npm. Il est possible de le personnaliser, par exemple en extrayant le CSS dans un fichier séparé, en utilisant un préprocesseur CSS, en masquant la sortie JS, etc.
```

Comment:
Vu que `package` n'est pas traduit, y-a-t'il du sens à traduire `to package` ?

Reply:
Dans ce genre de situation, je pense que ne pas traduire, ou du moins traduire en partie nous offre des options qui sont pas très jolies à lire du type **minimums pour packager votre**. Toutefois, si d'autres façons d'écrire ceci existent, pourquoi pas les utiliser !

Reply:
Oui je vois, après on a #4 pour discuter de ça, mais peut-être que @Haeresis est déjà tombé sur ce cas et pourrait peut-être nous aider :D